faqs.md

```diff
@@ -0,0 +1,7 @@
+# Frequently asked questions (FAQs)
+
+## How can I receive news and updates about the election?
+
+Communications that need to reach the wide community will be shared on the NumFOCUS Blog, Newsletter, relevant mainling lists, and social media channels.
+
+This GitHub repository and issue tracker will have all real-time activities.
```

Comment:
```suggestion
This GitHub repository and issue tracker will have all real-time activities.

## Can Election Committee member also nominate themselves?
```
I have no idea about this answer.

Reply:
I'm also not sure - let's ask in the meeting today?

Reply:
We asked it :)
```suggestion
This GitHub repository and issue tracker will have all real-time activities.

## Can Election Committee members also nominate themselves?
Yes they can.
```